packages/core/src/character.ts

```diff
@@ -127,8 +127,10 @@ export class Character {
         (object: Object3D) => {
           this.model = object as Object3D;
           this.preprocessModel(this.model);
+          this.model.scale.x = this.model.scale.y = this.model.scale.z = 0.01;
```

Comment:
Is this the correct convention / transformation? Or just accounting for some particular model?

Reply:
There is no convention AFAIK. This specific scale accounts approximately for a particular variety of models. It works well for most of the FBX models I tested from Mixamo, but even among those, there is variance.

One viable alternative would be to include a `modelScale` parameter as part of the [CharacterDescription](https://github.com/mml-io/mml-playground/blob/a1e091c719a97ae716550f4e02f1de89897b7cb1/packages/core/src/character.ts#L20)

Reply:
@MarcusLongmuir The latest commit implements the alternative described above.